draft-ietf-quic-multipath.md

```diff
@@ -575,6 +575,14 @@ any of which it can use when sending packets on that path; see also {{Section 5.
 Usually, it is desired to provide at least one additional connection ID for
 all used paths, to allow for migration.
 
+Over a given path, both endpoints use Connection IDs associated to a given Path
+ID. To initiate a path, each endpoint needs to advertise at least one Connection ID
+for a given Path ID to its peer. Endpoints SHOULD NOT create Path ID discontinuity
```

Comment:
```suggestion
for a given Path ID to its peer. Endpoints SHOULD NOT use Path ID discontinuity
```

Reply:
Maybe "Endpoints SHOULD NOT introduce discontinuity in the usage of Path ID through their Connection ID advertisements"?